multi-agent/adapters/inbound/flask/endpoints/builtins.py

```
@@ -0,0 +1,414 @@
+    regardless of the caller's identity. Used by the Repository Management
+    admin panel.
+    """
+    svc = current_app.container.resources_service
+    try:
+        resources = svc.find_all_builtins(category=category, type=type)
+        return jsonify({
+            "resources": [doc.model_dump(mode="json") for doc in resources],
+        }), 200
+    except ValueError as e:
+        return jsonify({"error": str(e)}), 400
+    except Exception as e:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)